core-storage/src/main/java/org/apache/kylin/storage/gtrecord/GTCubeStorageQueryBase.java

```diff
@@ -272,6 +274,22 @@ private FunctionDesc findAggrFuncFromCubeDesc(FunctionDesc aggrFunc) {
         return resultD;
     }
 
+    private long getQueryFilterMask(Set<TblColRef> filterColumnD) {
+        long filterMask = 0;
+
+        logger.info("Filter column set for query: " + filterColumnD.toString());
```

Comment:
Set.toString() will not return joined string by default, have a check?

Reply:
And will filterColumnD be null? If yes, NPE might happen here.

Reply:
HashSet.toString() will return like [,,...]

filterColumnD will not be null. Since it's initialized by 
Set<TblColRef> filterColumnD = Sets.newHashSet();